plugins/Kaleidoscope-LED-Stalker/src/kaleidoscope/plugin/LED-Stalker.cpp

```diff
@@ -71,6 +69,11 @@ void StalkerEffect::TransientLEDMode::update() {
 
   for (auto key_addr : KeyAddr::all()) {
     uint8_t step = map_[key_addr.toInt()];
+
```

Comment:
@tremby would you mind adding a  comment here explaining what's happening?

Reply:
Sure.

Reply:
I added a comment. Honestly I am not 100% certain that `live_keys` and `Key_Inactive` mean exactly what I think they do, so please do review! I am not well versed in the Kaleidoscope API.